Supertext.Base/Caching/IDecentralizedCache.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Threading.Tasks;
+using System.Threading;
+
+namespace Supertext.Base.Caching;
+
+public interface IDecentralizedCache
```

Comment:
Shouldn't this interface and its internal implementation be mentioned on an Autofac module somewhere?

Reply:
I do it in the extension method with the service collection. It is being used for configuring StackExchange.Redis anyway. Therefore it would be extra overhead with Autofac.
Like: `serviceCollection.AddScoped<IDecentralizedCache, RedisCache>();`